easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.5.3-iompi-2020a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'ParallelIO'
+version = '2.5.3'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'iompi', 'version': '2020a'}
+
+source_urls= ['https://github.com/NCAR/ParallelIO/releases/download/pio2_5_3/']
+sources = ['pio-2.5.3.tar.gz']
+checksums = ['63ae300d7b2e8790a272d442cb2856ff053810a05a9c37d28228fc4ebde1f5f5']
+
+
+dependencies = [
+    ('PnetCDF', '1.9.0'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('hdf5-mpi', '1.10.6'),
+    ('openmpi', '4.0.3'),
```

Comment:
We never have MPI, Compiler or Cuda dependencies. Those are part of the toolchains.